debian/rules

```diff
@@ -2,12 +2,17 @@
 
 export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 export DEB_LDFLAGS_MAINT_APPEND := -Wl,--as-needed
+include /usr/share/dpkg/buildflags.mk
```

Comment:
Is this required?  From reading https://wiki.debian.org/HardeningWalkthrough#debhelper_9 I thought the DEB_BUILD_MAINT_OPTIONS was sufficient.